cardboard/templates/reactbase.html

```diff
@@ -0,0 +1,60 @@
+{% load static %}
```

Comment:
made a dupe that removes some of the mandatory markup that we can't control from react.